doc/bootstrapping.txt

```diff
@@ -44,11 +44,11 @@ Firstly, common.Bootstrap:
    - this step will go away soon, or at least become provider specific
 
 The finalizer function, is run after the following checks from bootstrap.Bootstrap:
- * selects tools from the previously calculated set based on the architecture and series
+ * selects agent binaries from the previously calculated set based on the architecture and series
    of the instance that the provider started
- * makes sure that the tools are available
+ * makes sure that the agent binaries are available
```

Comment:
make sure

I realize all of these could be intended to say "this is what the finalizer function is doing" but for me it actually reads "this is a check that is being performed", so I think all of them read better as a list of actions than a list of  'foo selects'.

select agent binaries
make sure the agent binaries are available
create the machine config struct
set the requested agent binaries in that structure to the ones bootstrap knows about
(typo in bootstap)

I could be wrong, but try reading through it with singular vs plural and